File: laramie-comunicacion-webapi/src/domain/entities/lista.ts
export default class Lista {

    id: number;
	codigo: string;
	tipo: string;
	nombre: string;
	orden: number;

	constructor(
        id: number = 0,
		codigo: string = "",
		tipo: string = "",
		nombre: string = "",
		orden: number = 0
	)
	{
        this.id = id;
		this.codigo = codigo;
		this.tipo = tipo;
		this.nombre = nombre;
		this.orden = orden;
	}

	setFromObject = (row) =>
	{
        this.id = row.id ?? 0;
		this.codigo = row.codigo ?? "";
		this.tipo = row.tipo ?? "";
		this.nombre = row.nombre ?? "";
		this.orden = row.orden ?? 0;
	}

}

File: modules/interface/models/lista.py
from sqlalchemy import Column, BigInteger, String, Integer

import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
from shared.database import Base


class Lista(Base):
    __tablename__ = "interface_listas"

    id = Column(BigInteger, primary_key=True)
    codigo = Column(String(50), nullable=False)
    tipo = Column(String(50), nullable=False, index=True)
    nombre = Column(String(250), nullable=False)
    orden = Column(Integer, nullable=False, default=0)
